locales/en-US/strings.properties

```diff
@@ -1,10 +1,13 @@
 newtab_page_title=New Tab
 
 header_top_sites=Top Sites
+header_top_sites_upper=TOP SITES
```

Comment:
@flodolo any preference on the name? Not sure if it needs a separate localization note or people will know to try to make it uppercase if appropriate?

Reply:
String ID make sense, I think we can skip the localization note.